### DISCUSSION TITLE
exporting -> Extra Attributes

### DISCUSSION THREAD

**Discussion by MayaUser666** [Autodesk/maya-usd#1349] [Q&A]
How to carry Extra Attributes on the model when exporting usd in Maya2022?

The Extra Attributes have been prepared in the abc file I made before. After I imported Maya and exported usd, those Extra Attributes disappeared.

**Comment by kxl-adsk** [comment]
[This post](https://github.com/Autodesk/maya-usd/discussions/1304#discussioncomment-567816) outlines possible options for exporting custom attributes in general.

You also mentioned alembic and so it would be good for you to explore the `AlembicChaser` (see [docs](https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/commands/Readme.md#export-chasers-advanced)). One downside currently is that it's only available in the Autodesk plugin after loading the Pixar plugin (so you have to build this repository from sources).

Hope this helps,
KxL
